src/toolbox/setup/windows.ts

```diff
@@ -1,5 +1,190 @@
+
+interface Result {
+  success: boolean,
+  info?: string
+}
+
+enum RegType {
+  REG_SZ = "REG_SZ",
+  REG_EXPAND_SZ = "REG_EXPAND_SZ",
+  REG_DWORD = "REG_DWORD",
+  REG_QWORD = "REG_QWORD",
+  REG_MULTI_SZ = "REG_MULTI_SZ",
+  REG_DEFAULT = "REG_DEFAULT"
+}
+
+async function setEnv(name: string, value: string, type: RegType): Promise<Result> {
+  let result
+  try {
+    result = await regedit.promisified.list([`HKCU\\ENVIRONMENT`])
+  } catch (error) {
+    return {success: false, info: "Error while reading user environment variables from registry"}
+  }
+  
+  try {
+    if (result) {
+      let reg = {
+        "HKCU\\ENVIRONMENT": {}
+      };
+      (reg as any)["HKCU\\ENVIRONMENT"][name] = {
+        value,
+        type
+      }
+      await regedit.promisified.putValue(reg)
+      return {success: true}
+    }
+  } catch (error) {
+    return {success: false, info: "Error while saving environment variable to registry"}
+  }
+  return {success: false}
+}
+
+async function addToPath(BIN_PATH: string): Promise<Result> {
+  let result
+  try {
+    result = await regedit.promisified.list([`HKCU\\ENVIRONMENT`])
+  } catch (error) {
+    return {success: false, info: "Error while reading User Path from registry"}
+  }
+
+  try {
+    if (result) {
+      const pathString = result["HKCU\\ENVIRONMENT"].values.Path.value
+
+      const tokens = pathString.split(";")
+      let found = false
+      for (let x of tokens)
+        if (x === BIN_PATH)
+          found = true
+  
+      if (found) {
+        return {success: true, info: "Moddable BIN already in User PATH."}
+      } else {
+        const value = {
+          "HKCU\\ENVIRONMENT": {
+            "Path": {
+              value: pathString + ";" + BIN_PATH,
+              type: "REG_EXPAND_SZ"
+            }
+          }
+        }
+  
+        await regedit.promisified.putValue(value)
+        return {success: true}
+      }
+    }
+  } catch (error) {
+    return {success: false, info: `Error setting registry value`}
+  }
+  return {success: false}
+}
 
 export default async function (): Promise<void> {
-  print.warning('Windows setup is not currently supported')
+  const BIN_PATH = filesystem.resolve(
+    INSTALL_PATH,
+    'build',
+    'bin',
+    'win',
+    'release'
+  )
+  const BUILD_DIR = filesystem.resolve(
+    INSTALL_PATH,
+    'build',
+    'makefiles',
+    'win'
+  )
+
+  print.info(`Setting up Windows tools at ${INSTALL_PATH}`)
+
+  // Check for Visual Studio CMD tools
+  try {
+    await system.exec('nmake /?')
+  } catch (error) {
+    print.error('Visual Studio 2022 Community is required to build the Moddable SDK: https://www.visualstudio.com/downloads/')
+    // @@TODO: need to check if perhaps VS.bat has not been run...
+    process.exit(1)
+  }
+  try {
+    await system.exec(`git --version`)
+  } catch (error) {
+    print.error('git is required to clone the Moddable SDK: https://git-scm.com/download/win')
+    process.exit(1)
+  }
```

Comment:
Same comment about using `system.which`:

```suggestion
  if (system.which('git') === null) {
    print.error(
      'git is required to clone the Moddable SDK: https://git-scm.com/download/win'
    )
    process.exit(1)
  }
```